package.json

```diff
@@ -10,9 +10,9 @@
     "ember-addon",
     "ember-cli-deploy-plugin"
   ],
-  "author": "",
+  "author": "jdberrocal1",
   "license": "MIT",
-  "repository": "adopted-ember-addons/ember-cli-deploy-new-relic-sourcemap",
+  "repository": "jdberrocal1/ember-cli-deploy-new-relic-sourcemap",
```

Comment:
Since we're merging this into the `adopted-ember-addons`, so we want to leave this as is?